tools/Environments/DevHome.Environments/Views/LandingPage.xaml

```diff
@@ -66,6 +66,7 @@
                 <Grid>
                     <Button
                         x:Uid="MoreOptionsButton"
+                        Padding="0"

```

Comment:
Alternatively, should this padding go on the HorizontalThreeDotsStyle?

Reply:
yea you're right. Forgot that I could just update the style. Updated it and works just as well, Thanks.